spell/spell-reviewer-goerli-checklist.md

```diff
@@ -135,37 +143,40 @@ Spell Actions:
+        * [ ] Comment explaining `val` formula (`Debt ceiling * [ (1 + RWA stability fee ) ^ (minimum deal duration in years) ] * liquidation ratio`) is present
+        * [ ] Accompanying comment above `bump` line in format `// XXM * 1.XX^X * X.XX as a WAD` corresponding to the `val` calculation formula (e.g. `// 15M * 1.03^2 * 1.00 as a WAD`) is present along with the calculation formula on the line above
+        * [ ] IF combining `val` of multiple RWA ilks being combined, `val` calculation is done once per ilk and added to make the total, with workings provided in code comments. The existing `val` value can be retrieved by calling `read()` on `PIP_RWAXX` and converting the result into decimal.
     * [ ] Poke `spotter` to pull in the new price
-* [ ] Payments
-  * [ ] Streams (`DssVest`)
-    * [ ] `DssVest` Interface is correct
-    * [ ] Ensure that `cap` > max new vest `tot`/`tau` otherwise file cap as well
-    * [ ] Timestampts match Doc (`bgn`, `fin`)
-    * [ ] CUs Addresses match Doc (`usr`)
-    * [ ] Amount matches Doc (`tot`, if decimals are present consider using `ether`)
-    * [ ] Vesting Duration matches Doc (`tau`)
-    * [ ] Cliff Duration matches Doc (`eta`)
-    * [ ] Restricted (by default)
-    * [ ] Manager match Doc (`mgr`, set to zero for DAI streams by default)
-    * [ ] IF [DssVestTransferrable](https://github.com/makerdao/dss-vest/blob/master/src/DssVest.sol#L463) is used
-      * [ ] Ensure `DssVestTransferrable` allowance is increased by new vesting delta (by approving the `transferrable` vest contract to allowance + new total amount streamed)
-  * [ ] CUs MKR Transfers
-    * [ ] Recipient Addresses match Doc
-    * [ ] Transfers Amounts match Doc
-    * [ ] Follows Previous Patterns
-  * [ ] Direct SB DAI Payment
-    * [ ] Recipient Addresses match Doc
-    * [ ] Payment Amounts match Doc
-    * [ ] Follow Previous Patterns
-  * [ ] Ensure Recipient Addresses match `addresses_wallets.sol`
+* [ ] SubDAO Content
+  * [ ] SubDAO SubProxy spell execution
+    * [ ] SubDAO spell address matches exec doc
+    * [ ] SubDAO spell deployer is in `addresses_deployers`
+    * [ ] Executed using `ProxyLike(SUBDAO_PROXY).exec(SUBDAO_SPELL, abi.encodeWithSignature("execute()"));`
+    * [ ] Execution is NOT delegate call
+    * [ ] Gas cost will not be too high (low level call gas estimation is not done by our scripts)
+  * [ ] Maker Core (main spell) SubDAO actions (i.e. operate in Pause Proxy `DelegateCall` context)
+    * [ ] No SubDAO contract being interacted with is authed on a core contract like vat, etc. (script this eventually)
+    * [ ] SubDAO contract licensing and optimizations do not matter (not strictly)
```

Comment:
SubDAOs are already excluded from the base case `External Contracts`

`* [ ] External Contracts Calls (Not SubDAOs, e.g. Starknet)`

You're correct that message `* [ ] SubDAO contract licensing and optimizations do not matter (not strictly)` isn't really a check but is a note for reviewers, otherwise the standard procedure for checking smart contracts is missing a step.

Agree with your concerns here and will find a better solution in the future, though for now this is the only area where licensing of subdao contracts is covered